src/PointwiseFunctions/GeneralRelativity/GeneralizedHarmonic/Python/Bindings.cpp

```diff
@@ -16,6 +17,13 @@ namespace GeneralizedHarmonic::py_bindings {
 namespace {
 template <size_t Dim>
 void bind_impl(py::module& m) {  // NOLINT
+  m.def(
+      "spatial_ricci_tensor",
+     static_cast<tnsr::ii<DataVector,Dim> (*)(
```

Comment:
Please run clang-format over the file. An easy way to do that is to run `git-clang-format HEAD~1` (this runs over the last commit)